projects/batfish/src/test/java/org/batfish/common/BatfishTestUtils.java

```diff
@@ -0,0 +1,94 @@
+package org.batfish.common;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.Map;
+import java.util.SortedMap;
+import javax.annotation.Nullable;
+import org.apache.commons.collections4.map.LRUMap;
+import org.batfish.config.Settings;
+import org.batfish.config.Settings.EnvironmentSettings;
+import org.batfish.config.Settings.TestrigSettings;
+import org.batfish.datamodel.Configuration;
+import org.batfish.datamodel.ConfigurationFormat;
+import org.batfish.datamodel.DataPlane;
+import org.batfish.datamodel.Interface;
+import org.batfish.datamodel.collections.BgpAdvertisementsByVrf;
+import org.batfish.datamodel.collections.RoutesByVrf;
+import org.batfish.main.Batfish;
+import org.junit.rules.TemporaryFolder;
+
+public class BatfishTestUtils {
+
+  private static Batfish _batfish;
+  private static Batfish _batfishConfigurations;
```

Comment:
drop these; return new instances every time instead. Of course individual tests or test suites may cache the return value ; but multiple different tests won't share static state.